### ISSUE TITLE
Support zooming in preview, because some output graphs is unreadable without zooming 

### ISSUE THREAD

**Issue by tungdaomissmpeu** [develar/d2-intellij-plugin#17]
As title, example I have a graph looks like this
<img width="1607" alt="d2_need_zoom" src="https://github.com/develar/d2-intellij-plugin/assets/146047295/c3d4f864-4058-493d-b451-e38af7b780ed">
It is really hard to see what is on the graph, even if I use a big, high resolution monitor.
So it would be nice if the previewer have a zoom button, or zoom on Ctrl + MouseScroll

**Comment by mvbukov** [comment]
I agree, zooming is needed. I have sequence diagrams which are rather long. Current preview is too small.
